tf2_ros/CMakeLists.txt

```diff
@@ -171,5 +171,6 @@ add_rostest(test/transform_listener_time_reset_test.launch)
 endif()
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
+ament_export_dependencies(rclcpp)
```

Comment:
tf2_ros depends on rclcpp, so we have to make sure to export the dependencies for downstream users.